src/notifier/notification_sender.py

```
@@ -0,0 +1,780 @@
+            html_file_path,
+        )
+
+
+def send_to_feishu(
+    stats: List[Dict],
+    failed_ids: List,
+    report_type: str,
+    new_titles: Optional[Dict] = None,
+    id_to_name: Optional[Dict] = None,
+    update_info: Optional[Dict] = None,
+    proxy_url: Optional[str] = None,
```

Ruff (0.14.8)

Unused function argument: `proxy_url`

(ARG001)